build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -18,366 +18,49 @@
 ###########################################################################
 # rtl8721csm_download.sh
```

Comment:
Requesting you to maintain consistency in the commit title and description.
I'd suggest modifying as: -
- rtk _will use_ build/configs/common_download.sh for downloading.
- rtl8721csm_download.sh _will contain_ board specific functions.

To indicate that these changes will come into effect once this commit is applied.

Reply:
Done.